packages/runtime/src/index.ts

```diff
@@ -14,6 +15,7 @@ export {
   Interpreter,
   Runtime,
   AccountStore,
+  LogicSig,
```

Comment:
do we need this ? can we not use algosdk type ?

Reply:
getting this `Module '"algosdk"' has no exported member 'LogicSig'. Did you mean to use 'import LogicSig from "algosdk"' instead?`

Reply:
yes, currently we have LogicSig in types/algo-sdk.

Reply:
no i mean i get  the error above while doing `import { LogicSig } from "algosdk"`

Reply:
could you please try with yarn build, this shouldn't give error because we are exporting logicsig from types/algo-sdk.
working fine on my system